docs/modules/globby.md

```diff
@@ -0,0 +1,46 @@
+# globby
+
+`globby` is a globbing library that presents [some behaviors](https://github.com/sindresorhus/globby#features)
+no other libraries have, mainly the handling of negated patterns and directory expansion. It can be replaced
+with lighter and faster alternatives.
+
+# Alternatives
+
+## tinyglobby
+
+`tinyglobby` can be used as an alternative in most cases.
+It doesn't include all `globby` options though.
+
+[Project Page](https://github.com/SuperchupuDev/tinyglobby)
+
+[npm](https://npmjs.com/package/tinyglobby)
+
+## fdir + picomatch
+
+If you don't rely on `globby`'s special features, you can just use
+`fdir` combined with `picomatch`.
+Its API is nothing like `globby`'s, but it does the job if configured properly.
+
+[`fdir` Project Page](https://github.com/thecodrr/fdir)
+
+[`fdir` on npm](https://npmjs.com/package/fdir)
+
+[`picomatch` Project Page](https://github.com/micromatch/picomatch)
```

Comment:
i wouldn't bother linking to picomatch since the fdir docs should mention it already in their section on globbing